src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs

```diff
@@ -48,7 +49,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_FileSystemTypes(CustomC
                     .AddHeader(Alignment.Left, label: "UnixMode", width: 10)
                     .AddHeader(Alignment.Right, label: "User", width: 10)
                     .AddHeader(Alignment.Left, label: "Group", width: 10)
-                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 16)
+                    // Until LastWriteTime prints as '{0:d} {0:HH}:{0:mm}', it will always be local date + 6 wide.
+                    // https://github.com/PowerShell/PowerShell/issues/19723
+                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: CultureInfo.CurrentCulture.DateTimeFormat.ShortDatePattern.Length + 6)
```

Comment:
Looks interesting but there are formats which will be broken like:
```powershell
Get-Date -Date "09/7/2025" -Format "g yyyy-MM-dd"
AD 2025-09-07
```

Full list unique formats you can get with:
```powershell
(Get-Culture -ListAvailable).DateTimeFormat.ShortDatePattern | Sort -Unique
```